### ISSUE TITLE
Not work when build with 1.14.0

### ISSUE THREAD

**Issue by zw963** [wyhaines/timeout.cr#2]
```
In src/translater/selenium/session.cr:23:5

 23 | timeout(seconds) do
      ^------
Error: instantiating 'timeout(Float64)'


In lib/timeout/src/timeout.cr:50:9

 50 | Fiber.timeout(seconds.seconds)
            ^------
Error: wrong number of arguments for 'Fiber.timeout' (given 1, expected 2)

Overloads are:
 - Fiber.timeout(timeout : Time::Span, select_action : Channel::TimeoutAction)
```

Thanks

**Comment by wyhaines** [comment]
Thanks.  I will take a look.

**Comment by zw963** [comment]
It caused by [this](https://github.com/crystal-lang/crystal/commit/c8ecd9339f64a72c83f76f7c65975856ba96e3b5) commit which merged in 1.14.0

**Comment by wyhaines** [comment]
#3 fixes this, though the fix is probably less than ideal.

It will need to be revisited in the future and a better fix applied.
